packaging/userspace/build_and_package.example.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Example build script. Tweak as needed.
+set -e
+set -x
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )" # directory containing this script
+
+ICOMMANDS_SRC_DIR="$(readlink -s -m "${ICOMMANDS_SRC_DIR:-"${SCRIPTDIR}/../.."}")" # icommands source directory
+ICOMMANDS_BLD_DIR="$(readlink -s -m "${ICOMMANDS_BLD_DIR:-"${ICOMMANDS_SRC_DIR}/../icommands_build/userspace"}")"
+
+IRODS_SRC_DIR="$(readlink -s -m "${IRODS_SRC_DIR:-"${ICOMMANDS_SRC_DIR}/../irods_source"}")" # irods source directory
+IRODS_BLD_DIR="$(readlink -s -m "${IRODS_BLD_DIR:-"${ICOMMANDS_BLD_DIR}/irods_build"}")" # irods build directory
+
+IRODS_BUILD_PREFIX="$(readlink -s -m "${IRODS_BUILD_PREFIX:-"${IRODS_BLD_DIR}/pfx"}")" # temporary install prefix
+
+QTY_CPUS="$(nproc --all)"
+QTY_JOBS="${QTY_JOBS:-"$((QTY_CPUS * 2 / 3))"}"
```

Comment:
for 1 CPU... QTY_JOBS becomes 0?

why 2/3?     not a big deal, just curious.

Reply:
I usually run about 14 or 16 on my system, which has 24 cores. The idea is to use most cores, but not all, so the user can still make use of their machine while the build happens.